spec.html

```diff
@@ -1030,7 +1030,13 @@ <h1>Mathematical Operations</h1>
     <emu-clause id="sec-value-notation">
       <h1>Value Notation</h1>
       <p>In this specification, ECMAScript language values are displayed in *bold*. Examples include *null*, *true*, or *"hello"*. These are distinguished from longer ECMAScript code sequences such as `Function.prototype.apply` or `let n = 42;`.</p>
-      <p>Values which are internal to the specification and not directly observable from ECMAScript code are indicated with a ~sans-serif~ typeface. For instance, a Completion Record's [[Type]] field takes on values like ~normal~, ~return~, or ~throw~.</p>
+    </emu-clause>
+
+    <emu-clause id="sec-identity">
+      <h1>Identity</h1>
```

Comment:
I wonder if this whole section should be part of the *ECMAScript Data Types and Values* chapter, not the *Algorithm Contentions* one - it's describing semantics of objects, not just a notational convention.